pandas/io/formats/format.py

```diff
@@ -179,15 +180,19 @@ def __init__(self, series, buf=None, length=True, header=True, index=True,
 
     def _chk_truncate(self):
         from pandas.core.reshape.concat import concat
+        min_rows = self.min_rows
         max_rows = self.max_rows
+        if max_rows and min_rows is None:
+            min_rows = max_rows
+        min_rows = min(min_rows, max_rows)
         truncate_v = max_rows and (len(self.series) > max_rows)
         series = self.series
         if truncate_v:
-            if max_rows == 1:
-                row_num = max_rows
-                series = series.iloc[:max_rows]
+            if min_rows == 1:
```

Comment:
can min_rows be 0? (None turns it off)

Reply:
In practice it turns it also off. Is that the expected behaviour? 
max_rows of 0 means let it fit the screen, so in principle we could do here the same

Reply:
this can just be

```
if min_rows:
  ...
```

Reply:
No, this is a specific path for `min_rows == 1`, there is some special logic for that (eg you don't concat two parts).

Note that this is existing code, I didn't change the logic, only the variable name max_rows with min_rows

Reply:
I simplified the logic here a bit, touching less of the existing code